.circleci/config.yml

```
@@ -17,7 +17,7 @@ jobs:
           MAX_HEAP_SIZE: 1300m
           HEAP_NEWSIZE: 512m
 
-      - image: scalarlabs/scalar-ledger:1.3.1
+      - image: scalarlabs/scalar-ledger:2.0.1
```

```suggestion
      - image: scalarlabs/scalar-ledger:2.0.2
```
Sorry, it's now 2.0.2. 2.0.3 might be coming soon as well...

---

Thank you so much !